### ISSUE TITLE
Hot reload should be even less verbose

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#6609]
This should be collapsed into one line when you hit "r":
```
Syncing files to device...                           96ms
Performing hot reload...                             1793ms
Reloaded 260 of 439 libraries.
```

**Comment by Hixie** [comment]
We have learnt from usability studies that the more verbose our logging, the less people read it, and this leads to them to missing things like "press r to hot reload" entirely.

**Comment by raju-bitter** [comment]
What should the single line look like?
`Hot reload: Syncing files to device (96ms) -> performing hot reload (1793ms) -> reloaded 260 of 439 libraries.`

**Comment by devoncarew** [comment]
Something like:

```
Reloading app...                             1793ms
```

**Comment by b-strauss** [comment]
`Syncing files (96ms), reloading 260 of 439 libraries (1793ms)`

**Comment by mit-mit** [comment]
CC @johnmccutchan

**Comment by mit-mit** [comment]
I don't think we need to separate the measures of the sync and reload steps, so suggest:

When the reload starts:
```
Hot reloading app...                              1793ms
```

When the reload completes:
```
Hot reloading app... (17 of 460 libraries)        1793ms
```

**Comment by johnmccutchan** [comment]
I like @mit-mit's suggestion.

**Comment by bkonyi** [comment]
If this is still a usability issue, feel free to reopen, but I think the current hot reload message isn't verbose at all.
